buildSrc/src/main/kotlin/org/cqfn/save/buildutils/DockerStackConfiguration.kt

```diff
@@ -67,8 +68,14 @@ fun Project.createStackDeployTask(profile: String) {
             }
         }
 
-        val useOverride = (findProperty("useOverride") as String?).toBoolean()
         val configsDir = Paths.get("${System.getProperty("user.home")}/configs")
+        val useOverride = (properties.getOrDefault("useOverride", "true") as String).toBoolean()
+        val composeOverride = File("$configsDir/docker-compose.override.yaml")
+        if (useOverride && !composeOverride.exists()) {
+            logger.warn("Can't use override configuration, because ${composeOverride.canonicalPath} doesn't exist")
```

Comment:
?
```suggestion
            logger.warn("`useOverride` option is set to true, but can't use override configuration, because ${composeOverride.canonicalPath} doesn't exist")
```